duckduckgo/iOS
# Unify Configuration Management

Task/Issue URL: https://app.asana.com/0/72649045549333/1203797041821336/f
Tech Design URL:
CC: @bwaresiak 

**Description**:
Use common API and Configuration modules located in BSK  for fetching privacy related assets.
Add payload validation.

**Steps to test this PR**:
1. Run tests
2. Check if app downloads assets.
3. Check if app sends etag in requests.
4. Check if app handles 304s.
5. Check if app stores the resources.

---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
**When ready for review, remember to post the PR in MM**
